packages/relay/src/routes/commentRoute.ts

```diff
@@ -83,4 +85,30 @@ export default (
                 res.json({ txHash })
             })
         )
+
+    app.get('/api/comment/commentHistory', async (req, res) => {
+        try {
```

Comment:
Do we need try catch here? Why not just throw the error directly. Open discussion to the benefit of this.

Reply:
fixed